rewriter/ChainedSig.cc

```diff
@@ -0,0 +1,40 @@
+#include "rewriter/ChainedSig.h"
+#include "ast/Helpers.h"
+#include "ast/treemap/treemap.h"
+
+using namespace std;
+namespace sorbet::rewriter {
```

Comment:
This file doesn't introduce any tests. When you add tests, can you be sure to add rewrite-tree exp file tests? Ideally mentioning all of the builder methods that could show up.

Reply:
Done! I was waiting for initial feedback before putting some tests together.